projects/igniteui-angular/src/lib/grids/filtering/excel-style/grid.excel-style-filtering.component.ts

```diff
@@ -599,6 +599,13 @@ export class IgxGridExcelStyleFilteringComponent implements OnDestroy {
 
         this.uniqueValues.forEach(element => {
             if (element !== undefined && element !== null && element !== '') {
+                if (this.column.dataType === DataType.String &&
+                    this.column.filteringIgnoreCase &&
+                    this.listData.length > 0 &&
+                    this.listData.map(el => el.label.toLowerCase()).indexOf(element.toLowerCase()) !== -1) {
+                        return;
+                    }
+
```

Comment:
This doesn't seems to be the best place to apply the `filteringIgnoreCase`, it is better to apply it in the `generateUniqueValues` method. Also this solution is not the best in terms of performance. You can see a performance comparison [here](https://stackoverflow.com/a/43046408) and an implementation for case insensitive scenarios [here](https://stackoverflow.com/a/48731445).